SPECS/qemu-dp.spec

```diff
@@ -82,6 +83,9 @@ install -m 644 qemu-dp-tracing "%{buildroot}%{_libdir}/qemu-dp/bin/qemu-dp-traci
 %{_libdir}/qemu-dp/bin
 
 %changelog
+* Mon Mar 4 2019 Ronan Abhamon <ronan.abhamon@vates.fr> - 2.12.0-1.10.0.1.xcp
```

Comment:
We add changelog entries only if we increase the release number, but here we don't need to increase it because we want to do more changes before building a new release of the RPM. But I wouldn't be upset if you increased the release number instead.